person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] We can save some code here. We are repeating ourselves. There is no need to elongate this method. We can simply have this.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```